deploy/eck-resources/values.yaml

```diff
@@ -0,0 +1,431 @@
+# TODO:
+# 1) Verify Openshift support
+# 2) Verify support of PSPs/SCCs
+
+# Global settings that will be applied to all resources (Elasticsearch/Kibana/etc)
+#
+global:
```

Comment:
Well, I was trying to avoid us having to manage a helm chart for each Elastic stack component, and keep things simpler with a single helm chart that supports all components (when their templates are added in a future release).  Really want other's opinions on single vs multiple helm charts for the individual components.

I added this `global` section (which could certain be renamed) so that you could have a set of labels/annotations that were applied to each resource managed by this instance of this chart, to easily identify the kibana/ES instances owned by this chart.